inginious/frontend/plugins/UN_template/footer.html

```diff
@@ -31,7 +31,9 @@
             <p class="col-sm-12 col-md-6 derechos">
                 © Copyright 2018<br/> Algunos derechos reservados.<br/>
                 <a title="Comuníquese con el administrador de este sitio web" href="mailto:uncode_fibog@unal.edu.co">uncode_fibog@unal.edu.co</a><br/>
-                <a href="/">Acerca de este sitio web</a><br/> Actualización:09/07/19
+                <a href="/">Acerca de este sitio web</a>
+                <br/><a href="$get_homepath()/data_policy">Política de tratamiento de datos personales</a>
+                <br/> Actualización:09/07/19
```

Comment:
This modification should not go here as this template is static and we only modify this plugin when the uni's template is updated and it is required to do so. For that, modify the same `UNCode` plugin, in the `uncode.js` file in the `static` folder. You will probably have to modify the `updateFooter` function on this file, and add the link to the data policy modifying this footer rather than UN template's footer.